spec/discovery_dispatcher/indexing_job_spec.rb

```diff
@@ -2,9 +2,8 @@
   describe '.perform' do
     it 'performs the indexing process successfully' do
       VCR.use_cassette('index_xz404nk7341') do
-        index_job = described_class.new('index', 'xz404nk7341', 'target1')
-        Rails.configuration.target_urls_hash = { 'target1' => { 'url' => 'http://localhost:3000' } }
-        expect_any_instance_of(DiscoveryDispatcher::IndexingJob).to receive('build_request_url').with('xz404nk7341', 'http://localhost:3000', 'target1').and_return('RestClient.put "http://localhost:3000/items/xz404nk7341?target1", ""')
+        index_job = described_class.new('index', 'xz404nk7341', 'searchworkspreview')
+        expect_any_instance_of(DiscoveryDispatcher::IndexingJob).to receive('build_request_url').with('xz404nk7341', 'http://www.example-indexer.com', 'searchworkspreview').and_return('RestClient.put "http://localhost:3000/items/xz404nk7341?target1", ""')
```

Comment:
why does the return value have "`RestClient.put`" in it? shouldn't it just be a URL?